_includes/footer.html

```diff
@@ -18,4 +18,7 @@
       </div>
     </div>
   </div>
-</footer>
\ No newline at end of file
+</footer>
+
+<script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>

```

Comment:
@anasdove2020 Would it possible to only use pure javascript? I am thinking we can reduce the usage of additional library.

Reply:
Sure, will use pure javascript.